server/utils/logger.js

```diff
@@ -103,7 +103,13 @@ try {
 } catch (error) {
   // no op
 }
-/* eslint-enable global-require */
-/* eslint-enable import/no-unresolved */
+
+export const logResponseError = (error) => {
+  if (error.response) {
+    const {status, data} = error.response;
+    if (status >= 500) Logger.error(error.response);
+    else Logger.info(`Request failed with ${status}: ${JSON.stringify(data)}`);
```

Comment:
I think we should also add URL of the reponse if available. Otherwise, we will be clueless what endpoint generated the error. Is this information already included in data?

Reply:
@codesankalp what about:
```
else Logger.info(`Request to ${url} failed with ${status}:\n${JSON.stringify(data)}`);
```